### DISCUSSION TITLE
404 install directory on fresh install - ubuntu 22.04

### DISCUSSION THREAD

**Discussion by spiffyfrodo** [forkcms/forkcms#3554] [Help] [answered]
I have done a fresh install of Ubuntu Server 22.04 (no adds like docker, etc.)
I have done several attempts of installing forkcms including manual, download, composer, etc.
Now I have followed this guide to a tea: https://www.howtoforge.com/how-to-install-fork-cms-in-ubuntu-server-22-04/
--> The only thing I've tried is changing the ServerName to localhost (I tried 127.0.0.1 and 127.0.0.1:80 ... both seem to work)
I get redirected to a PARENT directory ../install which does not exist
my .htaccess file is there...
Most solutions were about .htaccess

192.168.1.xxx = APACHE PAGE
192.168.1.xxx/forkcms redirects to 192.168.1.xxx/install
but no install folder is generated.
everything is www-data owned.

I really don't know how to get this install folder to materialise.

**Comment by tijsverkoyen** [comment]
I suspect mod_rewrite is not enabled.
Or `AllowOverride` is set to `None`.

**Reply by spiffyfrodo** [reply]
/var/www/html$ a2enmod rewrite
Module rewrite already enabled

according to howtoforge:

```
<VirtualHost *:80>
     ServerAdmin admin@example.com
     DocumentRoot /var/www/html/forkcms
     ServerName 127.0.0.1

     <Directory /var/www/html/forkcms/>
          Options FollowSymlinks
          AllowOverride All
          Require all granted
     </Directory>

     ErrorLog ${APACHE_LOG_DIR}/error.log
     CustomLog ${APACHE_LOG_DIR}/access.log combined

</VirtualHost>
```

Now I don't know if AllowOverride is also done for the parent directory?

**Comment by tijsverkoyen** [comment]
I'm not a server specialist. But probably there will be something in the logs why the .htaccess is not adhered.

**Reply by spiffyfrodo** [reply, marked as answer]
Thank you for your help so far. 

I had grav installed too as a test site and it too had 404 errors. I was able to take your intuition and apply it to the apache file at:

/etc/apache2/apache2.conf

and changed AllowOverride All for the home directory /var/www/html

This worked for grav but not for forkcms.

Then I changed /var/www/html/forkcmd -> /var/www/forkcms and renamed forkcms to html.

The new homepage worked. It had some errors but gave a link to install/ and after clicking it it generated the file.

Hopefully this will help anyone else with the issue.

Best.
